src/main/scala/chisel3/Driver.scala

```diff
@@ -94,12 +95,25 @@ object Driver extends BackendCompilationUtilities {
 
   def emit[T <: Module](ir: Circuit): String = Emitter.emit(ir)
 
-  def dumpFirrtl(ir: Circuit, optName: Option[File]): File = {
-    val f = optName.getOrElse(new File(ir.name + ".fir"))
-    val w = new FileWriter(f)
-    w.write(Emitter.emit(ir))
+  def dumpFirrtl(ir: Circuit, optName: Option[File], optAnnoName: Option[File] = None): File = {
```

Comment:
Is giving `optAnnoName` a default value too much of a dirty trick? Rocket calls a two argument version of `dumpFirrtl`